timeboost-sequencer/src/lib.rs

```diff
@@ -288,12 +287,15 @@ impl Task {
                 }
             }
 
-            // always sync DKG bundles
-            self.next_dkg(&mut candidates, &mut pending_dkgs);
-            if !pending_dkgs.is_empty() {
-                tracing::debug!(num_bundles = %pending_dkgs.len(), "enqueuing dkg bundles");
-                if let Err(err) = self.decrypter.enqueue_dkg(&mut pending_dkgs).await {
-                    error!(node = %self.label, %err, "dkg enqueue error");
+            if pending.is_none() {
+                while let Some(dkg) = dkg_bundles.pop_front() {
+                    if !self.decrypter.has_capacity() {
+                        dkg_bundles.push_front(dkg);
+                        break;
```

Comment:
1. why do we only process DKG bundles when `pending` is none? shouldn't we always try to make progress on processing DKG bundles?
2. what's the benefit of imposing the capacity on DKG processing? Say the decrypter don't have capacity for `worker_rx` receiving inclusion list anymore, why should this prevents the decrypter from sending Dkg command to the worker? 
(maybe checking the `self.decrypter.worker_tx.capacity()>0` is reasonable imo)